tests/test_predictor.py

```diff
@@ -20,9 +20,24 @@ def test_boston_dataset():
     gradients = y_train.astype(np.float32)
     hessians = np.ones(1, dtype=np.float32)
 
+    min_samples_leaf = 5
+    max_leaf_nodes = 31
     grower = TreeGrower(X_train_binned, gradients, hessians,
-                        min_samples_leaf=7, max_leaf_nodes=31)
+                        min_samples_leaf=min_samples_leaf, max_leaf_nodes=max_leaf_nodes)
     grower.grow()
+
+    import pytest
+    lb = pytest.importorskip("lightgbm")
+    est_lightgbm = lb.LGBMRegressor(n_estimators=1,
+                                    min_data_in_bin=1,
+                                    learning_rate=1,
+                                    min_data_in_leaf=min_samples_leaf,
+                                    num_leaves=max_leaf_nodes)
+    est_lightgbm.fit(X_train_binned, y_train)
+
+    from pygbm.plotting import plot_tree
+    plot_tree(grower, est_lightgbm)
```

Comment:
Please do not put such plots in the regular tests. Use the `examples/` folder for visual debugging instead.

Reply:
Also if you want to compare lightgbm and pygbm on the Boston dataset, please add a new test in `test_compare_lightgbm.py` instead.

Reply:
Sure, this is not meant to stay, I just left it so you can reproduce my plots if you wanted to